packages/instrumenter/src/mutators/boolean-literal-mutator.ts

```diff
@@ -9,12 +11,31 @@ import { NodeMutator } from './index.js';
 export const booleanLiteralMutator: NodeMutator = {
   name: 'BooleanLiteral',
 
-  *mutate(path) {
-    if (path.isBooleanLiteral()) {
-      yield types.booleanLiteral(!path.node.value);
-    }
-    if (path.isUnaryExpression() && path.node.operator === '!' && path.node.prefix) {
-      yield deepCloneNode(path.node.argument);
+  *mutate(path, level: MutationLevel | undefined) {
+    if (level === undefined) {
+      // No mutation level selected, allow everything.
+      if (path.isBooleanLiteral()) {
+        yield types.booleanLiteral(!path.node.value);
+      }
+      if (path.isUnaryExpression() && path.node.operator === '!' && path.node.prefix) {
+        yield deepCloneNode(path.node.argument);
+      }
+    } else if (level.BooleanLiteral !== undefined) {
+      // Mutation level has been set and the included BooleanLiterals are defined
+      if (path.isBooleanLiteral()) {
+        const value: boolean = path.node.value;
+        if (
+          (value && level.BooleanLiteral.some((lit) => lit == 'TrueToFalse')) ||
+          (!value && level.BooleanLiteral.some((lit) => lit == 'FalseToTrue'))
```

Comment:
And you are using loose equality instead of strict equality

Reply:
I'm not sure whether that'd make things much prettier since it's not applicable when the other conditional is true (`path.isUnaryExpression() && path.node.operator === '!' && path.node.prefix`), so you can't just pass `path.node.value` for example. But I'll see what I can do.

Reply:
Changed

Reply:
Looks good!